docs/equality.md

```diff
@@ -78,18 +78,18 @@ Test the equality of structures nested within arrays:
 
 ```ts
 import { array, eq } from "fp-ts";
+import { Eq } from "fp-ts/lib/Eq";
 
 type Point = {
   x: number;
   y: number;
 };
 
-const eqPoint: eq.Eq<Point> = eq.getStructEq({
-  x: eq.eqNumber,
-  y: eq.eqNumber,
-});
+const eqPoint: Eq<Point> = {
+  equals: (p1: Point, p2: Point) => p1 === p2 || (p1.x === p2.x && p1.y === p2.y)
+}
 
-const eqArrayOfPoints: eq.Eq<Array<Point>> = array.getEq(eqPoint);
+const eqArrayOfPoints = array.getEq<Point>(eqPoint);
```

Comment:
Does this also work without providing the generic variable? Same as further up, I would hope that it is not necessary anymore.

Reply:
I'll check.

Reply:
Removed.